src/snowcli/cli/snowpark/procedure/commands.py

```diff
@@ -218,20 +221,23 @@ def procedure_update(
         execute_as_caller,
         install_coverage_wrapper,
     )
+    return OutputData()
 
 
 @app.command("package")
+@with_output
 def procedure_package(
     pypi_download: str = PyPiDownloadOption,
     check_anaconda_for_pypi_deps: bool = CheckAnacondaForPyPiDependancies,
     package_native_libraries: str = PackageNativeLibrariesOption,
-):
+) -> OutputData:
     """Packages procedure code into zip file."""
     snowpark_package(
         pypi_download,  # type: ignore[arg-type]
         check_anaconda_for_pypi_deps,
         package_native_libraries,  # type: ignore[arg-type]
     )
+    return OutputData()
```

Comment:
How does it works? First we will do `print_db_cursor` and later empty output?

Reply:
This command don't print cursor and `Output()` prints `Done`

Btw I removed `print_db_cursor`